src/utils/i18n.ts

```diff
@@ -19,11 +19,11 @@ const DEFAULT_LANGUAGE_CODE = 'en';
 
 /**
  * Gets a message for a given language code, from a map of messages.
- * @param {!Object<string, string>} map
- * @param {?string|?Element} languageCodeOrElement
- * @return {?string}
  */
-export function msg(map, languageCodeOrElement) {
+export function msg(
+  map: {[key: string]: string},
```

Comment:
I think the value type should be `string?` since in general `map[key]` can be undefined.